modules/pruning-images-manual.adoc

```diff
@@ -5,20 +5,91 @@
 [id="pruning-images-manual_{context}"]
 = Manually pruning images
 
-The Pruning Custom Resource enables automatic image pruning. However, administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits by running the following command:
+The Pruning Custom Resource enables automatic image pruning. However, administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
-[source,terminal]
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
+* Expose the image registry.
+
+.Procedure
+
+Running image pruning as a `Job` or `CronJob` on the cluster.
+
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits create a YAML file for the `pruner` service account, for example:
```

Comment:
Do we need _To manually prune images that are no longer required by the system due to age, status, or exceed limits_?
Just: _Create a YAML file for the `pruner` service account, for example:_ ?
Do you need to `oc create` the YAML?